### DISCUSSION TITLE
Is notarization required for sidecar after code signing to run in distribution?

### DISCUSSION THREAD

**Discussion by cloudsolace** [tauri-apps/tauri#7387] [Q&A] [answered]
I have an app with a sidecar which i am code signing manually and copying it over before notarization of actual app. I keep getting **Exception Type:        EXC_BAD_ACCESS (SIGKILL (Code Signature Invalid))** for the sidecar, i am not sure whats the issue. Do i need to notarize the sidecar too?

**Comment by FabianLars** [comment, marked as answer]
Yes it is, and tauri's bundler currently does not take care of it itself. I apologize if i wasn't clear enough with marking https://github.com/tauri-apps/tauri/issues/7381 as a duplicate of that other issue. So let me be clear: Until tauri's bundler explicitly supports this (= the linked issue gets resolved) you will have to sign&notarize the sidecar yourself first.

**Reply by cloudsolace** [reply]
Thanks @FabianLars
